project/src/components/sorting-form/sorting-form.tsx

```diff
@@ -1,18 +1,74 @@
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { SortType } from '../../const';
+import { changeSortType } from '../../store/action';
+import { useState, useRef } from 'react';
+import { useOnClickOutside } from 'usehooks-ts';
+import cn from 'classnames';
+
 function SortingForm(): JSX.Element {
+  const [isOptionsOpen, setOptionsView] = useState<boolean>(false);
```

Comment:
Явно указывать тип не следует для примитивов тс прекрасно сам выведет
```suggestion
  const [isOptionsOpen, setOptionsView] = useState(false);
```